src/main/java/racingcar/model/RacingCars.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.model;
+
+import racingcar.RandomNumberGenerator;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+
+    private static final String CAR_POSITION_KEY = "position";
+    private static final String CAR_NAME_KEY = "name";
+
+    private static final int MIN_FORWARD_NUMBER = 4;
+
+    private int maxPosition = Integer.MIN_VALUE;
+    private final List<Car> racingCars;
+
+    public RacingCars(List<Car> racingCars) {
+        this.racingCars = racingCars;
+    }
+
+    public void forwardTry() {
+        for (Car racingCar : racingCars) {
+            int randomNumber = RandomNumberGenerator.generate();
+            if (randomNumber >= MIN_FORWARD_NUMBER) {
```

Comment:
```suggestion
            if (canMove()) {
```
메서드로 따로 빼면 가독성이 더 좋을 것 같아요

Reply:
이 부분은 생각 못했었는데 다른 분 코드 보면서도 깨닫게 되었습니다!
감사합니다! ㅎㅎ 😃